DESCRIPTION

```diff
@@ -26,7 +26,8 @@ Imports:
     jsonlite,
     stats,
     git2r (>= 0.11.0),
-    withr
+    withr,
+    rhub
```

Comment:
Good point, I will do this, too. It is pretty lightweight, except for packages that are only needed by devtools: `httr` and `curl`. But if `testthat` is not required, then `rhub` should not be, either.